### PR TITLE
docs: update troubleshooting in wireguard section

### PR DESCRIPTION
**Pull request by Tim-Anderson**
Signed-off-by: Tim Anderson <tim.anderson@foundries.io>

## Readiness

* [x ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

Many customers have had problems setting up servers.

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [x ] Avoid changing any header associated with a link/reference.
* [x ] Step through instructions (or ask someone to do so).
* [x ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 99637cb6acfca841593293b3377386f94aded4b6 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1558/docs/artifacts/html/index.html

**Review by doanac — COMMENTED** [review]
lgtm

File: source/reference-manual/remote-access/wireguard.rst [hunk 1]
```diff
@@ -140,6 +140,18 @@ Troubleshooting
...
+.. note::
+
+  When configuring a server behind a firewall make sure the desired port
+  is passed through to the host running the server.
+
+When trying to activate the server you get::
+
+  ERROR: A UDP socket is already opened on 165.227.222.126:5555
+
+After making sure no other service is using that port, add ``--no-check-ip``
```
Comment (kprosise) [inline source/reference-manual/remote-access/wireguard.rst:152]:
```suggestion
Make sure no other service is using that port, then add ``--no-check-ip``
```

"...making sure no other service is using the port" Should be given as an imperative/instruction. 

This is also a bit ambiguous... Do you mean that any service using the port needs to be closed first, and then run the command with the `--no-check-ip` flag? It is also unclear what command this is being added to, though I'm assuming wireguard given the context.

**Review by kprosise — COMMENTED** [review]
Suggestions made. A couple things were unclear, but I may be missing context. We also have the separate Troubleshooting document, though feel free to leave it here if that makes more sense. Let me know if you have any questions.

File: source/reference-manual/remote-access/wireguard.rst [hunk 2]
```diff
@@ -140,6 +140,18 @@ Troubleshooting
 Wireguard uses UDP. This can be difficult to troubleshoot. A very common problem
 is when the VPN server has a firewall blocking traffic to the Wireguard port.
 
+.. note::
+
+  When configuring a server behind a firewall make sure the desired port
```
[the remark this answers is not in the document]
Comment (Tim-Anderson) [reply source/reference-manual/remote-access/wireguard.rst:145]: Done

File: source/reference-manual/remote-access/wireguard.rst (hunk 1 above)
Comment (Tim-Anderson) [reply source/reference-manual/remote-access/wireguard.rst]: I separated the instruction from the corrective action

**Comment by doanac** [comment]
Docs for 4d9a01d636701a9bc6d3a54221ead96f27dedbff are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1561/docs/artifacts/html/index.html

**Review by kprosise — APPROVED** [review]
LGTM! I will go ahead and merge.

**Pull request merged**